packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -163,9 +163,21 @@ export interface AllowedTypeMetadata {
 	 */
 	readonly custom?: unknown;
 
-	// TODO metadata for enablable types will be added here
+	/**
+	 * If defined, indicates that an allowed type is enablable. Before upgrade, this type can be loaded into the tree from a document
```

Comment:
Same note about "writable" from the changeset applies here.

Saying it may or may not be allowed based on enablement via the stored schema would make more sense.

Would also be good to clarify if enableable types are allowed in unhydrated trees (and error on hydration), or if they are errors to put in unhydrated trees.